doc/appendices/command-line/traffic_layout.en.rst

```diff
@@ -60,6 +60,12 @@ Options:
 
  Force create sandbox and overwrite existing directory 
 
+ 4. relocate flag for creating: ::
+
+    traffic_runroot --relocate --init /path/to/sandbox
```

Comment:
Is there a `traffic_runroot` command? I looked at the build but I didn't see it.